.gitignore

```diff
@@ -13,3 +13,4 @@ yarn-error.log
 .env
 .changelog
 coverage/
+lerna-debug.log
```

Comment:
remove

Reply:
why this should be in vcs? does it harm?

Reply:
it doesn't belong to the scope of the PR.

Reply:
its missing